app/models/concerns/hud_concerns/client.rb

```diff
@@ -60,6 +60,17 @@ def race_multi
       end
     end
 
+    # This can be used to retrieve numeric representations of the client's race
```

Comment:
```suggestion
    # This can be used to retrieve numeric representations of the client's race
    # @return [Array<Integer>] An array of integers representing the client's race where the race field is `1` with any valid values of `RaceNone` 
```